cypress/e2e/euronews/tests.cy.js

```diff
@@ -0,0 +1,135 @@
+/// <reference types="Cypress" />
+import newsletters from '../../fixtures/newsletters.json';
+
+describe('Euronews tests', () => {
+    beforeEach(() => {
+        // Step 1
+        // Follow the euronews.com 
+        cy.visit('https://www.euronews.com/');
+    });
+
+    it('test case 1', () => {
+        // Main page of Euronews is opened 
+        cy.url().should('eq', 'https://www.euronews.com/');
+        cy.get('#didomi-notice-agree-button').click();
+        // Top tags are displayed
+        cy.get('[class*="tag-list"]').should('be.visible');
+        // Views displayed 
+        cy.get('.b-topstories-home').should('be.visible');
+        cy.get('.b-topstories-home__aside__article h2 a').eq(0).then(element => {
```

Comment:
Beside hardcodes.

You can cy.get('.b-topstories-home__aside__article h2 a').eq(0).click().invoke('text').then(homeTitle => {
cy.get('.jsArticleFirst.swiper-slide-active h1').should('contain.text', homeTitle.trim());
})